models/apiLibrary.models.js

```diff
@@ -0,0 +1,15 @@
+const fs = require('fs/promises')
+
+
+const fetchApiJSON = () => {
+        return fs.readFile(`endpoints.json`, 'utf-8')
+        .then((data) => {
+            return JSON.parse(data)
+        })
+        .catch((err) => {
```

Comment:
you don't need to error handle in both the model and controllers, next in the controller will pick up any errors from the model and pass them to your middleware

Reply:
resolved :)